src/components/dropdowns/_dropdowns.scss

```diff
@@ -21,7 +20,7 @@ $include-html: false !default;
     border-radius: $dropdownHeight / 2;
     min-height: $dropdownHeight;
     height: $dropdownHeight;
-    cursor: pointer;
+    cursor: pointer;    
```

Comment:
Everything that needs JS to work correctly, like this dropdown (toggling between open/closed requires JS), needs `cursor: pointer`. Everything that just works natively can't have `cursor: pointer`. So we are fine here.